docs/en/Contributing-Getting-Started.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: global
+title: Contributor Getting Started
+nickname: Contributor Getting Started
+group: Resources
+---
+
+Welcome to the Alluxio community! We are excited for your contributions and engagement in the
+community! Here is a guide on how to get started with becoming a member in the Alluxio open source
+community.
+
+## Requirements
+
+The main requirement is a computer with Mac OSX or Linux installed. Alluxio does not have Windows support at the moment.
+
+## Software and Account Preparation
+
+There are several accounts and setup required before contributing to Alluxio.
+
+### Java 7 or 8
+
+Developing Alluxio requires Java 7 or 8. If you are unsure of which version of java you have, you
+can check with this command:
+
+```bash
+$ java -version
+```
+
+If you do not have java installed, download and install [the java SDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html).
+
+### Maven
+
+Alluxio uses Maven to manage the build process. If you do not have Maven already installed, you can
+[download Maven](https://maven.apache.org/download.cgi), and
+[install Maven](https://maven.apache.org/install.html). 
+
+### Git
+
+The Alluxio open source project uses the Git Distributed Version Control System for its source code.
+Therefore, Git needs to be installed to contribute code to Alluxio.
+
+If you do not already have `git`, [install git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+
+### GitHub Account
+
+The Alluxio open source code base is hosted on GitHub in the [Alluxio repository](https://github.com/Alluxio/alluxio).
+
+You will need a GitHub account in order to contribute to Alluxio. If you do not already have a
+GitHub account, please [create a GitHub account](https://github.com/).
+
+You will also need to know which [email addresses you have associated with your GitHub account](https://help.github.com/articles/adding-an-email-address-to-your-github-account/).
+
+### Jira Account
+
+The [Alluxio open source project uses JIRA](https://alluxio.atlassian.net/projects/ALLUXIO/issues)
+to track features, bugs, and issues.
+[Please create a Alluxio JIRA account](https://alluxio.atlassian.net/projects/ALLUXIO/issues) in
```

Comment:
Link to https://tachyon.atlassian.net/admin/users/sign-up instead

nit: move "please" outside the link text so that the link part of the text is a verb phrase